runner/src/check_todo.rs

```diff
@@ -15,16 +15,27 @@
 //
 
 use super::*;
+use regex::Regex;
 
 /// A [`Runnable`] command that checks for the existence of todos in the codebase with no associated
 /// GitHub issue number.
 pub struct CheckTodo {
     path: String,
+    todo_pattern: Regex,
 }
 
 impl CheckTodo {
     pub fn new(path: String) -> Box<Self> {
-        Box::new(CheckTodo { path })
+        Box::new(CheckTodo {
+            path,
+            todo_pattern: Regex::new("TODO\\(#[0-9][0-9]*\\)").expect("Could not parse regex"),
```

Comment:
Drive-by: `\d+`

Also, there might be stuff of interest in #1007&hellip;

Reply:
Wow! Nice. What is your plan for #1007? I could not go to sleep without a regex here, but I feel better now knowing that there is #1007 😄

Reply:
I *might* get back to #1007 now that `runner` is more stable, but it will need a bunch of merging and I'm unlikely to get to it in the short term, so I'd suggest carrying on with your plan.  

(I may also just drop #1007 &ndash; the core idea of checking online issue status is quite brittle because the GitHub API rate limits kick in quickly).